requirements-test.txt

```diff
@@ -3,8 +3,7 @@ numpy>=1.19.5 ; python_version <= '3.9'
 numpy>=1.21.6 ; python_version == '3.10'
 numpy>=1.23.5 ; python_version >= '3.11'
 scikit-learn==1.3.2
-pandas==2.0.3 ; python_version == '3.8'
-pandas==2.1.2 ; python_version >= '3.9'
+pandas==2.1.2
```

Comment:
There is no built pandas 2.1.2 for py38 in PyPI.

Reply:
Seems odd that the bot would've suggested the change then, but if so I guess the PR should be closed